tests/integration/agent_factory.py

```diff
@@ -216,3 +216,36 @@ def kubernetes_agent(memory_local_cache, workspace: Workspace):
     )
 
     return agent
+
+@pytest.fixture
+def get_nobel_prize_agent(
+    agent_test_config, memory_local_cache, workspace: Workspace
+):
+    command_registry = CommandRegistry()
+    command_registry.import_commands("autogpt.commands.file_operations")
+    command_registry.import_commands("autogpt.app")
+
+    ai_config = AIConfig(
+        ai_name="Get-PhysicsNobelPrize",
+        ai_role="An autonomous agent that specializes in physics history.",
+        ai_goals=[
+            "Write to file the winner's name(s), affiliated university, and discovery of the 2010 nobel prize in physics. Write your final answer to 2010_nobel_prize_winners.txt.",
```

Comment:
@merwanehamadi, my understanding was that these were integration tests to ensure autogpt is working over time. Yes, the information is known.  When I ask the challenge a question to ChatGPT with their beta Web Browsing enabled, challenge a is answered as well. I can update this to be static information that happened after Sept 2021. Is this what you are thinking for the challenge?

Reply:
@merwanehamadi , yes this is known information. When running challenge A in ChatGPT with the Web Browser beta on, it gets the answer correct as well. This test was intended to focus on the integration test of pulling data that will not change over time. It can be made more complicated, additional information retrievals, by asking for more details about Andre and Konstantin. 

If needed, I could change it to be non-changing information post Sept 2021, but I am not sure what that gets us. I am open to ideas.